### ISSUE TITLE
Windows 10 / Java 8 issue ?

### ISSUE THREAD

**Issue by elabree** [phweda/MFM#18]
Hi,

The tool crashes on my windows 10 computer. Tried the latest and 2 earlier releases.
- Java 8 Update 261 (64-bit)
C:\Users\Egon>java -version
java version "1.8.0_261"
Java(TM) SE Runtime Environment (build 1.8.0_261-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.261-b12, mixed mode)

This seems to be the latest java release. But the manual mentiones 10 and 11 ????
16 GB RAM installed

[MFM_ERRout-260942.txt](https://github.com/phweda/MFM/files/5146935/MFM_ERRout-260942.txt)

Can you tell me what I am doing wrong ?

Thanks,

Egon.

**Comment by phweda** [comment]
Did you get the Settings dialog and input your paths?
Did you unzip the release as is with the file \<MFM Root>_/Data/MFM_MAME_0.198.zip_ ?
By crash do you mean it just hangs? 

FYI your Java is fine. Only use the latest release.

If you retry make sure you do a clean re-install.

**Comment by elabree** [comment]
Hi,
Thanks for the quick response.
Yes I unzipped the release MFM_0.9.500.7z as-is.
I did get the settings dialog
After "Loading Data" the program just closed itself

It seems I was off track finding the issue. It was not java (thanks for confirming that). When I tried to do a clean re-install I noticed that 7-zip showed a warning about the header being corrupt. 

So I did a fresh download of the release and now it works beautifully. Thanks for your hard work and for publishing it.
Well done & sorry to bother you with this.

Regards,

Egon.
